editor/project_settings_editor.cpp

```diff
@@ -586,7 +586,7 @@ ProjectSettingsEditor::ProjectSettingsEditor(EditorData *p_data) {
 	general_editor->add_child(header);
 
 	property_box = memnew(LineEdit);
-	property_box->set_placeholder(TTR("Select a setting or type its name"));
+	property_box->set_placeholder(TTR("Select a Setting Or Type Its Name"));
```

Comment:
I don't remember if `or` and `its` are stop words, and whether they should be capitalized. But this looks more natural to me:

```suggestion
	property_box->set_placeholder(TTR("Select a Setting or Type its Name"));
```